json.ml

```diff
@@ -7,6 +7,7 @@ let show_json = Settings.add_bool("show_json", false, `User)
 
 (* Type synonyms *)
 type handler_id = int
+type maybe_connection_url = string option
```

Comment:
(1/2) Why not just make `maybe_connection_url` a synonym of `string`? ... [ continued below ]